src/main/java/com/sybit/airtable/Airtable.java

```diff
@@ -182,9 +184,14 @@ public void setProxy(String proxy) {
 
         this.config.setProxy(proxy);
         if (proxy == null) {
-            Unirest.setProxy(null);
+            Unirest.config().reset();
+            Unirest.config().proxy(null);
         } else {
-            Unirest.setProxy(HttpHost.create(this.config.getProxy()));
+            HttpHost httpHost = HttpHost.create(this.config.getProxy());
+            String hostName = httpHost.getHostName();
+            int port = httpHost.getPort();
+            Unirest.config().reset();
+            Unirest.config().proxy(hostName, port);
```

Comment:
Unclear whether this would be an issue for a proxy that uses https as the scheme.

Reply:
Unirest seems to ignore scheme at this point: https://kong.github.io/unirest-java/#proxies